src/components/DataQuery.js

```diff
@@ -35,36 +36,35 @@ class DataQuery extends React.Component {
   }
 
   fetchData(query) {
-    this.setState({
-      loading: true,
-      error: false
-    });
+    this.setState({ loading: true });
```

Comment:
think it is a shame to remove error from state. Having it there makes it possible for the rendered Component to show an error when there is one. Without it, the component (in this case the DataTable) have no way of knowing that the api failed. When the API fails, then the spinner shouldn't show and the table shouldn't show. When that happens it is most useful for the user to see an error component instead. See also https://github.com/gbif/registry-console/issues/21#issuecomment-447846093